support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -2,35 +2,12 @@
 import type { Tests } from './abtest';
 
 // ----- Tests ----- //
-export type AmazonPaySingleUSTestVariants = 'control' | 'amazonPay' | 'notintest';
 export type StripePaymentRequestButtonScaTestVariants = 'control' | 'sca' | 'notintest';
 
 const contributionsLandingPageMatch = '/(uk|us|eu|au|ca|nz|int)/contribute(/.*)?$';
-const usContributionsLandingPageMatch = '/us/contribute(/.*)?$';
 const digitalLandingPageMatch = '/(uk|us|eu|au|ca|nz|int)/subscribe/digital(/.*)?$';
 
 export const tests: Tests = {
-  amazonPaySingleUS2020: {
-    type: 'OTHER',
-    variants: [
-      {
-        id: 'control',
-      },
-      {
-        id: 'amazonPay',
-      },
-    ],
-    audiences: {
-      ALL: {
-        offset: 0,
-        size: 1,
-      },
-    },
-    isActive: true,
-    referrerControlled: false,
-    seed: 13,
-    targetPage: usContributionsLandingPageMatch,
```

Comment:
Does removing this mean that it will now appear on all pages - I think we still only want this on the US page. Would it be more appropriate to increase the audience size?

Reply:
You might need to add a country filter criteria to checkouts.js

Reply:
We do not load amazon pay for non-US users: https://github.com/guardian/support-frontend/blob/master/support-frontend/assets/helpers/paymentIntegrations/amazonPay.js#L74